### ISSUE TITLE
Update the wiki

### ISSUE THREAD

**Issue by coreycaitlin** [18F/doi-extractives-data#1184]
...to reflect the current status and link to key resources (including the research branch).

**Comment by coreycaitlin** [comment]
Questions for @meiqimichelle:

- Is the [problem statement](https://github.com/18F/doi-extractives-data/wiki/Problem-Statement) up-to-date? Has anything changed?
- Do you envision the [hypotheses](https://github.com/18F/doi-extractives-data/wiki/Hypotheses) staying as-is, or should we revisit and assess?
- Is the [IA](https://github.com/18F/doi-extractives-data/wiki/Information-Architecture) still useful as a wiki page, or have we built all this info into the live site? If the latter, can we deprecate this wiki page?
- Okay to remove [narrative content](https://github.com/18F/doi-extractives-data/wiki/Narrative-Content) outline? This seems to have moved to the IA page.

**Comment by meiqimichelle** [comment]
- The problem statement needs to be updated with results from the last quarter and launch. Maybe this is easiest to do after #1188.
- We need to revisit the hypotheses, too. They could stand for another round of [LeanUX-style brainstorming](https://docs.google.com/document/d/14iYigu1cFd_SJrVt0rz3QvqIoGxQaTryDRCmUjmLS84/edit) to see what we've learned over the last few months.
- I think the IA is still useful, but more as a record of how we mapped to the exec summary. Maybe we can rename it to something that more resembles a styleguide and move it to a less prominent spot in the navigation. Thoughts?
- Yes!!
